backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/document/impl/DocTypeDAO.java

```diff
@@ -343,4 +346,20 @@ public Optional<String> getCounterDocBaseTypeByDocBaseType(@NonNull final String
 			return Optional.ofNullable(counterDocBaseTypeByDocBaseType.get(docBaseType));
 		}
 	}
+
+	@Cached
+	@Override
+	public DocType loadById(final DocTypeId id)
+	{
+		return fromDB(getById(id));
+	}
+
+	private DocType fromDB(final I_C_DocType docType)
```

Comment:
pls make it static

tomato tomato: usually i am calling this method `fromRecord` but i can live with `fromDB` :)

Reply:
done.